gradio/outputs.py

```diff
@@ -49,50 +49,6 @@ def deserialize(self, x):
         return x
 
 
-class Textbox(OutputComponent):
```

Comment:
If you delete this, and someone does the following import:

`from gradio.components import Textbox` 

Does this work fine?

Reply:
If not, we should do something similar to what I have `inputs.Textbox` above where we construct one class from the other.

But if these imports all work, then I agree it would be better to avoid it

Reply:
I am not in favor of having different initilizations for the input or output, I think that makes combining them meaningless and would prefer having a NewComponent type for blocks instead of this.
People will be able to reference Textbox through components, inputs and outputs. And use it both as an input or output.

However if are going to create code duplication by having different classes for input and output let's have NewComponents and don't break anything that Interface uses.

If we are going to combine components input and output should merge into one.


These are my thoughts, WDYT?